core/arch/arm/mm/mobj.c

```diff
@@ -155,12 +157,6 @@ struct mobj *mobj_phys_alloc(paddr_t pa, size_t size, uint32_t cattr,
 	enum teecore_memtypes area_type;
 	void *va;
 
-	if ((pa & CORE_MMU_USER_PARAM_MASK) ||
```

Comment:
Okay, then I returned back this check in v2 delta. 
I'll add the new `mobj` type in different PR if you don't mind.